[2022-11-29 14:08:24,383] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: vaccine_dags.transformData manual__2022-11-29T14:08:21.751178+00:00 [queued]>
[2022-11-29 14:08:24,396] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: vaccine_dags.transformData manual__2022-11-29T14:08:21.751178+00:00 [queued]>
[2022-11-29 14:08:24,396] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-11-29 14:08:24,396] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-11-29 14:08:24,396] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-11-29 14:08:24,410] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): transformData> on 2022-11-29 14:08:21.751178+00:00
[2022-11-29 14:08:24,415] {standard_task_runner.py:52} INFO - Started process 718 to run task
[2022-11-29 14:08:24,419] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'vaccine_dags', 'transformData', 'manual__2022-11-29T14:08:21.751178+00:00', '--job-id', '1190', '--raw', '--subdir', 'DAGS_FOLDER/vaccine.py', '--cfg-path', '/tmp/tmpf9e_688z', '--error-file', '/tmp/tmp2f3lx3oi']
[2022-11-29 14:08:24,420] {standard_task_runner.py:77} INFO - Job 1190: Subtask transformData
[2022-11-29 14:08:24,477] {logging_mixin.py:109} INFO - Running <TaskInstance: vaccine_dags.transformData manual__2022-11-29T14:08:21.751178+00:00 [running]> on host acb48b0a6e0d
[2022-11-29 14:08:24,529] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vaccine_dags
AIRFLOW_CTX_TASK_ID=transformData
AIRFLOW_CTX_EXECUTION_DATE=2022-11-29T14:08:21.751178+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-29T14:08:21.751178+00:00
[2022-11-29 14:08:24,588] {python.py:175} INFO - Done. Returned value was: None
[2022-11-29 14:08:24,708] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=vaccine_dags, task_id=transformData, execution_date=20221129T140821, start_date=20221129T140824, end_date=20221129T140824
[2022-11-29 14:08:24,752] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-11-29 14:08:24,792] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
